.prettierignore

```diff
@@ -19,6 +19,8 @@
 AppXManifest.xml
 copyright-header.txt
 dist/
+docs/art/ada-cat.ansi256.txt
+docs/art/ada-cat.ascii.txt

```

Comment:
ada-cat.ascii.txt no longer required here and in copyright-header.config.json below?

Reply:
Yes. Fixed in new commit. Thanks.